graph-proxy/src/graphql/workflow_templates.rs

```diff
@@ -246,11 +235,11 @@ impl TryFrom<argo_workflows_openapi::IoArgoprojWorkflowV1alpha1ClusterWorkflowTe
                 .metadata
                 .annotations
                 .remove("workflows.argoproj.io/description"),
-            arguments: ArgumentSchema::new(
+            arguments: Json(ArgumentSchema::new(
                 workflow_template.spec,
                 &mut workflow_template.metadata.annotations,
-            )?,
-            ui_schema: UiSchema::new(&mut workflow_template.metadata.annotations)?,
+            )?),
+            ui_schema: UiSchema::new(&mut workflow_template.metadata.annotations)?.map(Json),
```

Comment:
just out of curiosity: why are you using `Json()` in the first case and `().map(Json)` in the second?

Reply:
In the first instance `ArgumentSchema::new` returns `ArgumentSchema` so can be simply wrapped to give `Json<ArgumentSchema>`, in the second instance `UiSchema::new` returns `Option<UiSchema>` hence we use `::map` to apply the method to the inner value giving `Option<Json<UiSchema>>` (wrapping would give `Json<Option<UiSchema>>` which we don't want)